### DISCUSSION TITLE
Catppuccin theme looks weird

### DISCUSSION THREAD

**Discussion by Pix3lexe** [catppuccin/tmux#386] [Help] [answered]
I installed catppuccin plugin on my PC a long time ago and it always looked the same. On a new machine I installed it through my dotfiles and now it looks weird:
![image](https://github.com/user-attachments/assets/3f75f04f-9249-4f04-91d9-dcef34f7fede)

This is my .tmux.conf
![image](https://github.com/user-attachments/assets/2cea85b0-30c8-4002-844e-51d72ad7f968)

**Machine:**
Arch Linux
Desktop: GNOME, Wayland
Terminal: Kitty

Using JetBrainsMono NerdFont

**Comment by kjnsn** [comment, marked as answer]
Check out https://github.com/catppuccin/tmux/issues/291 for advice on how to upgrade from earlier versions.

And a word of warning: kitty may break tmux, you are kind of on your own.
